docs/altInstances.md

```diff
@@ -26,6 +26,23 @@ var flux = new MyAlt();
 
 # AltClass
 
+## AltClass#constructor
+
+> constructor(config: object) : Alt
+
+The alt constructor takes an optional configuration object. This is where you can configure your alt instance.
+
+```js
+var flux = new MyAlt({
+  dispatcher: new MyDispatcher()
+});
+```
+### Config Object
+
+The following properties can be defined on the config object:
+
+- *dispatcher*: By default alt uses Facebook's Flux [dispatcher](https://github.com/facebook/flux/blob/master/src/Dispatcher.js), but you can provide your own dispatcher implementation for alt to use. Your dispatcher must have a similar interface to the Facebook dispatcher including, `waitFor`, `dispatchToken`, `register`, and `dispatch`.
```

Comment:
My bad on the issue, `dispatchToken` is something that exists on the store not on the dispatcher itself. So we can remove this :)